### PR TITLE
Style changes and updates so that the main docs pages pass Vale linting with no warnings or errors

### PR DESCRIPTION
**Pull request by discdiver**
- Minor updates for style and fixes some typos.
- Primary docs pages pass Vale linting with no warnings or errors.
- Adds Vale exclusions to some docs sections.

### Checklist

- [X] This pull request includes a label categorizing the change e.g. `maintenance`, `fix`, `feature`, `enhancement`, `docs`.
- [ ] This pull request references any related issue by including "closes `<link to issue>`"
  
  - If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [ ] If this pull request adds new functionality, it includes unit tests that cover the changes
- [ ] If this pull request removes docs files, it includes redirect settings in `mint.json`.
- [ ] If this pull request adds functions or classes, it includes helpful docstrings.

### CODE REVIEW

File: docs/3.0rc/automate/events/webhook-triggers.mdx [hunk 1]
```diff
@@ -259,7 +259,7 @@ identifier, so `body.friendly-name` will not work. Use `body['friendly-name']` i
 those cases.
 </Tip>
 
-Prefect Cloud will attempt to parse any other content type (such as `text/plain`) as if it were JSON first.
+Prefect Cloud will attempt to parse any other media type (such as `text/plain`) as if it were JSON first.
```
Comment (billpalombi) [inline docs/3.0rc/automate/events/webhook-triggers.mdx:262]: In this case `Content-Type` is the literal name of the field, so we should either update the linter so it doesn't apply here or make it `Content-Type`

**Review by billpalombi — COMMENTED** [review]
One nit, otherwise looks good!

File: docs/3.0rc/automate/events/webhook-triggers.mdx (hunk 1 above)
Comment (discdiver) [reply docs/3.0rc/automate/events/webhook-triggers.mdx]: Done.

File: docs/3.0rc/manage/cloud/index.mdx [hunk 2]
```diff
@@ -19,7 +19,7 @@ The **Manage accounts** subsection covers aspects of Prefect Cloud accounts:
 - [Manage API keys](/3.0rc/manage/cloud/manage-users/api-keys/) explains how to grant access to Prefect Cloud from a local execution environment.
 - [Configure single sign-on](/3.0rc/manage/cloud/manage-users/configure-sso/) describes how (SSO) authentication integration is supported through identity providers with OIDC and SAML. Directory Sync and SCIM provisioning is also available. <span class="badge enterprise"></span>
 - [Audit Cloud activity](/3.0rc/manage/cloud/manage-users/audit-logs/) shows how Prefect provides a record of user activities to monitor for security and compliance. <span class="badge pro"></span> <span class="badge enterprise"></span>
-- [Object Access Control Lists](/3.0rc/manage/cloud/manage-users/object-access-control-lists/) (ACLs) allow administrators to restrict deployment and block access to individual users. <span class="badge enterprise"></span> 
+- [Object access control lists](/3.0rc/manage/cloud/manage-users/object-access-control-lists/) (ACLs) allow privlidged users to restrict deployment and block access to individual users. <span class="badge enterprise"></span>
```
Comment (billpalombi) [inline docs/3.0rc/manage/cloud/index.mdx:22]:
```suggestion
- [Object access control lists](/3.0rc/manage/cloud/manage-users/object-access-control-lists/) (ACLs) allow privileged users to restrict deployment and block access to individual users. <span class="badge enterprise"></span> 
```

**Review by billpalombi — APPROVED** [review]

**Pull request merged**